module/renderer/src/commonMain/kotlin/org/cru/godtools/shared/renderer/content/RenderText.kt

```diff
@@ -1,28 +1,68 @@
 package org.cru.godtools.shared.renderer.content
 
 import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
 import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
 import androidx.compose.ui.text.font.FontStyle
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.unit.dp
+import coil3.compose.AsyncImage
+import coil3.compose.LocalPlatformContext
+import coil3.request.ImageRequest
 import com.github.ajalt.colormath.extensions.android.composecolor.toComposeColor
 import org.cru.godtools.shared.renderer.content.extensions.textAlign
+import org.cru.godtools.shared.renderer.content.extensions.toPath
 import org.cru.godtools.shared.renderer.content.extensions.visibility
 import org.cru.godtools.shared.renderer.state.State
+import org.cru.godtools.shared.renderer.util.LocalResourceFileSystem
+import org.cru.godtools.shared.tool.parser.model.Resource
 import org.cru.godtools.shared.tool.parser.model.Text
 
 @Composable
-internal fun ColumnScope.RenderText(text: Text, state: State) = RenderTextNode(
-    text,
-    Modifier
+internal fun ColumnScope.RenderText(text: Text, state: State) = Row(
+    modifier = Modifier
         .visibility(text, state)
         .padding(horizontal = Horizontal_Padding)
-        .fillMaxWidth(),
-)
+        .fillMaxWidth()
+) {
+    val imagePaddingToText: Int = 10
```

Comment:
let's change this to 8: https://github.com/CruGlobal/godtools-android/blob/develop/ui/base-tool/src/main/res/layout/tool_content_text.xml#L25

Reply:
and move this to be a constant defined outside of the composable, similar to: https://github.com/CruGlobal/kotlin-mpp-godtools-tool-parser/blob/feature/compose/module/renderer/src/commonMain/kotlin/org/cru/godtools/shared/renderer/content/RenderParagraph.kt#L12

Reply:
`SCREAMING_SNAKE_CASE` was the standard format for constants in Java, which Kotlin had adopted. Recently I've been seeing compose constants defined in `CamelCase`, especially in compose libraries from Google.

For now either casing is fine for the constants (assuming ktlint doesn't complain). If in the future we decide on a specific pattern to follow we can clean up the style then